src/scanner/Scanner.ts

```diff
@@ -100,8 +100,13 @@ export class Scanner {
     this.d(`Components (${projectComponents.length}):`, inspect(projectComponents));
     const practicesWithContext = await this.detectPractices(projectComponents);
     this.d(`Practices (${practicesWithContext.length}):`, inspect(practicesWithContext));
-    await this.fix(practicesWithContext);
-    const practicesAfterFix = await this.detectPractices(projectComponents);
+
+    let practicesAfterFix;
```

Comment:
Is it `any` either after line `107`?

Reply:
It can be PracticeWithContext[] or undefined, will fix it